osbuild/pipeline.py

```diff
@@ -119,7 +119,7 @@ def description(self, *, with_id=False):
             description["id"] = self.id
         return description
 
-    def run(self, tree, runner, build_tree, monitor, libdir, output_dir=None, var="/var/tmp"):
+    def run(self, tree, runner, build_tree, monitor, libdir, output_dir, var="/var/tmp"):
```

Comment:
You can get rid of the `output_dir` conditional below as well.

Reply:
Apart from that redundant conditional, this looks good!

Reply:
Adjusted, thanks!